latest/docs-ref-conceptual/release-notes-azure-cli.md

```diff
@@ -16,6 +16,66 @@ ms.assetid: ce0428f7-0a59-4e72-9237-d907b171af51
 
 # Azure CLI 2.0 release notes
 
+## September 22, 2017
+
+Version 2.0.18
+
+### Resource
+
+* Added support for showing built-in policy definitions
+* Added support mode parameter for creating policy definitions
+* Added support for UI definitions and templates to `managedapp definition create`
+* BREAKING CHANGE: managedapp: Update to latest ARM package, which includes changing resource type from appliances to applications and applianceDefinitions to applicationDefinitions
```

Comment:
@derekbekoe - I'm not sure if this is an actual user-facing change or not. Couldn't create a test managed app to check. Can you remove or improve this line?

Reply:
done